en/drafts/originals/agent-based-model-communication-networks.md

```diff
@@ -661,35 +665,33 @@ plt.title("Distribution of Letters Sent by Agents")
+{% include figure.html filename="en-or-agent-based-model-communication-networks-06.png" alt="Histogram where the y-axis shows the number of agents, and the x-axis shows how many letters were sent by those agents. The numbers of letters sent range from about 25 to over 50." caption="Figure 6. Histogram of letters sent by agents after 100 steps of simulating the model." %}
 
-You can also use `pandas` to export the data to a CSV (comma separated value) file, which can be opened by any common spreadsheet application or opened by `pandas`.
+You can also use `pandas` to export the data to a CSV ([comma-separated value](https://en.wikipedia.org/wiki/Comma-separated_values)) file, which can be opened by any common spreadsheet application.
 
-If you do not specify a file path, the file will be saved in the local directory. After you run the code below you will see a file appear (*agent_data.csv*)
+If you do not specify a file path, the file will be saved in the local directory. After you run the code below, you will see the file `agent_data.csv` appear.
 
 ```python
 agent_letters.to_csv("agent_data.csv")
 ```
 
-Having exported the data we can then apply several approaches to test the hypotheses that we encoded in the model. The goal to systematically test - or validate - a model is to check if the model actually represents what it is supposed to. This can range from simple testing of your expectations versus the outputs, to analyzing the internal consistency of the model, over a detailed exploration of the possible parameters of your simulation (a so-called parameter space), to a detailed calibration to available empirical data. Mehdizadeh et al 2022, p. 8-9, coming from the discipline of mobility studies, offer a sensible differentiation of various validation methods as well as a good example into how Agent-based models are evaluated in other fields.[^23]
+Having exported the data, we can now follow several approaches to test the hypotheses that we initially encoded in the model. The goal to systematically test - or validate - a model is to check if the model actually represents what it is supposed to. This can range from simply comparing your expectations to the outputs; to analyzing the internal consistency of the model over a detailed exploration of the possible parameters of your simulation (a so-called 'parameter space'); all the way to a detailed calibration to available empirical data. Mehdizadeh et al 2022, p. 8-9, writing from the perspective of mobility studies, offer a sensible differentiation of various validation methods, as well as good examples of how to evaluate Agent-Based Models in other fields.[^23]
```

Comment:
I struggled a little with the sentence that starts 'This can range from [...].' Especially, I wasn't sure what you meant by 'analyzing the internal consistency of the model over a detailed exploration of the possible parameters of your simulation', or 'a detailed calibration to available empirical data'.

I imagine these are complex methods which are difficult to describe in just a few words, but would you be able to try rephrasing them slightly, to make them a little clearer? It might help to break that long sentence up.

Reply:
```suggestion
Having exported the data, we can now follow several approaches to test the hypotheses that we initially encoded in the model. The goal to systematically test - or validate - a model is to check if the model actually represents what it is supposed to. This can range from simply comparing your expectations to the outputs to systematically analyzing the internal consistency of the model. Calibrating your model to available empirical data is one of the most simple and robust methods of testing, but of course this might be challenging due to data availability. A more involved method is to explore the effects of a wide range of possible parameters of your simulation in detail (a so-called 'parameter space').  Mehdizadeh et al 2022, p. 8-9, writing from the perspective of mobility studies, offer a sensible differentiation of various validation methods, as well as good examples of how to evaluate Agent-Based Models in other fields.[^23]
```

Reply:
I restructured and extended this section a bit and hope this is a bit more clear now - you're totally right that this was hard to read before.

Reply:
Thank you very much!